theano/gof/type.py

```diff
@@ -997,10 +1019,26 @@ def __init__(self, *args, **kwargs):
             type(self).__name__ + ': expected 0 or only 1 extra parameter "ctype".'
         ctype = kwargs.pop('ctype', 'int')
```

Comment:
I use double as default ctype for base class EnumType (to allow constants to have double values), but I use int as default ctype for class EnumList, as here the constants are integers in any case.